src/RazorLight/RazorLight.csproj

```diff
@@ -75,4 +85,11 @@
       <PackagePath></PackagePath>
     </None>
   </ItemGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.AspNetCore.Html.Abstractions" Version="2.2.0" />
+    <PackageReference Include="Microsoft.AspNetCore.Razor" Version="2.2.0" />
+    <PackageReference Include="Microsoft.AspNetCore.Razor.Design" Version="2.2.0" />
+    <PackageReference Include="Microsoft.AspNetCore.Razor.Runtime" Version="2.2.0" />
+  </ItemGroup>
```

Comment:
this does not look like a good idea at all
trying to get portability at the price of depending on Obsolete / possibly CVE affected package

Probably all consumer having CVE check would have to opt out from `RazorLight`

At the same time introducing possible crash for consumer using AspNetCore on netcoreapp3.x / net5.0 / net6.0 if they actively needs new API since 3.x from Types inside the following assemblies:
```
Microsoft.AspNetCore.Html.Abstractions
Microsoft.AspNetCore.Razor
Microsoft.AspNetCore.Razor.Design
Microsoft.AspNetCore.Razor.Runtime
```

introducing this change would then also make lots of people jump out the boat for that reason.
if you want to hard pin on `2.2.0` I suggest you create a TFM for `netcoreapp2.1` or `netcoreapp2.2` and you stick to that

The cost of `FrameworkReference` is, unless I'm mistaken, just space disk on the output artifact, instead of creating possible `MissingMethodException` here.

The alternative is to split this repo is `RazorLight` and `RazorLight.WithoutAspNetCore` something like that